internal/controller/manager/rbac.go

```diff
@@ -0,0 +1,139 @@
+package manager
+
+/*
+Copyright 2020 Crunchy Data Solutions, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	"text/template"
+
+	"github.com/crunchydata/postgres-operator/internal/config"
+	"github.com/crunchydata/postgres-operator/internal/ns"
+
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// ErrReconcileRBAC defines the error string that is displayed when RBAC reconciliation is
+	// enabled for the current PostgreSQL Operator installation but the Operator is unable to
+	// to properly/fully reconcile its own RBAC in a target namespace.
+	ErrReconcileRBAC = "operator is unable to reconcile RBAC resource"
+)
+
+// ReconcileRBAC is responsible for reconciling  a configMap resource that has obtained from
+// the ConfigMap controller's worker queue
+func (c *ControllerManager) reconcileRBAC(targetNamespace string) {
+
+	log.Debugf("Controller Manager: Now reconciling RBAC in namespace %s", targetNamespace)
+
+	// Use the image pull secrets of the operator service account in the new namespace.
+	operator, err := c.controllers[targetNamespace].kubeClientset.CoreV1().
+		ServiceAccounts(c.pgoNamespace).Get(ns.OPERATOR_SERVICE_ACCOUNT, metav1.GetOptions{})
+	if err != nil {
+		log.Errorf("%s: %v", ErrReconcileRBAC, err)
+	}
```

Comment:
Do we want to return here?

Reply:
Not returning here so that we can still try to reconcile RBAC that isn't dependent on the operator ServiceAccount, e.g. the Roles and RoleBindings.

Reply:
@andrewlecuyer Cool. Can you please add a comment around that?

Reply:
Sure thing!